src/control/server/config/server_test.go

```diff
@@ -407,6 +407,70 @@ func TestServerConfig_updateServerConfig(t *testing.T) {
 	}
 }
 
+func TestServerConfig_MDonSSD_Constructed(t *testing.T) {
```

Comment:
Did this come in with a weird merge? I don't have a problem with including it if it passes... Just surprising to see it in this patchset.

Reply:
This was added to master in 41191f84319a69a9e361522fa91f094a91bf8a4c so shouldn't be appearing in this change set.

Reply:
@kjacque can we try to figure out why it is appearing here before we land please?

Reply:
It showed up with the merge because I had to resolve a conflict with master. If you check out the full patch with all commits, it doesn't show up as a difference.